app/src/main/res/layout/activity_main.xml

```diff
@@ -6,14 +6,17 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         tools:context=".MainActivity">
-
-    <TextView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="Hello World!"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintLeft_toLeftOf="parent"
-            app:layout_constraintRight_toRightOf="parent"
-            app:layout_constraintTop_toTopOf="parent"/>
-
+    <SearchView
+            android:layout_width="370dp"
```

Comment:
Do we want static widths and heights here?

Reply:
hah, that was actually auto-generated by Android Studio - I'm trying out the new tools I hadn't used before. Apparently they're not perfect. Let me fix that for you!